src/api/maximize.js

```diff
@@ -1,9 +1,41 @@
 import OS from 'os-family';
 import findWindow from './find-window';
-import { execFile } from '../utils/exec';
+import { execFile, exec } from '../utils/exec';
 import BINARIES from '../binaries';
 
+function getBoundsFromString (boundsString) {
+    var boundsArray = boundsString.split(', ');
 
+    return {
+        array:  boundsArray,
+        left:   boundsArray[0],
+        top:    boundsArray[1],
+        right:  boundsArray[2],
+        bottom: boundsArray[3]
+    };
+}
+
+async function getWindowBounds (windowId, bundleId) {
+    return getBoundsFromString(await exec(`osascript -e 'tell app id "${bundleId}" to get bounds of window id ${windowId}'`));
```

Comment:
It's hard to read when you use await function call as an argument for another function. Please do it separately